CharlieBackend.AdminPanel/Controllers/AccountController.cs

```diff
@@ -48,9 +44,11 @@ public ViewResult Login()
         [HttpPost]
         public async Task<IActionResult> Login(AuthenticationDto authDto)
         {
-            var httpResponseToken = await _apiUtil.SignInAsync($"{_config.Value.Urls.Api.Https}/api/accounts/auth", authDto);
+            var httpResponseToken = await _apiUtil.SignInAsync($"api/accounts/auth", authDto);
+
+            httpResponseToken = httpResponseToken.Replace("Bearer ", "");
```

Comment:
Use Replace on 47 line.

Reply:
Solved at this [commit](https://github.com/ita-social-projects/WhatBackend/pull/362/commits/8c1d6cc0c405f1cd3b93324b26e2a4a14e43cb96)